custom/icds_reports/views.py

```diff
@@ -444,6 +444,9 @@ def get(self, request, *args, **kwargs):
         }
 
         config.update(get_location_filter(location, domain))
+        # query database at same level for which it is requested
+        if config.get('aggregation_level') > 1:
```

Comment:
actual change of this PR

Reply:
nit: it's good to pass some default value in the get function to avoid TypeError while comparing between None and int, (in case the aggregation_level key is not present).

Reply:
hey, it would always have 'aggregation_level' key, `get_location_filter` function appends that to it. and above where config is initialized that is also initialized with 'aggregation_level' key.

Reply:
cool, good to go